packages/react-native/template/README.md

```diff
@@ -0,0 +1,76 @@
+This is a [**React Native**](https://reactnative.dev) project bootstrapped with [`npx react-native@latest init`](https://github.com/react-native-community/cli/tree/main#creating-a-new-react-native-project).
+
+# Getting Started
+
+>**Note:** Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions till [Creating a new application](https://reactnative.dev/docs/environment-setup#:~:text=Creating%20a%20new%20application) step, before proceeding.
+
+## Step 1: Start the Metro Server
+
+First, you will need to start Metro, the JavaScript _bundler_ that ships _with_ React Native. Metro "takes in an entry file and various options, and returns a single JavaScript file that includes all your code and its dependencies."—[Metro Docs](https://facebook.github.io/metro/docs/concepts)
+
+To start Metro, run `yarn start` _inside your React Native project folder_:
+
+```bash
+yarn start
+```
+
+Doing so, will start the **Metro Bundler**.
+
+## Step 2: Start your Application
+
+Let Metro Bundler run in its _own_ terminal. Open a _new_ terminal _inside your React Native project folder_ and run the following:
+
+```bash
+# For Android
+yarn android
+
+# For iOS
+yarn ios
+```
+
+If everything is set up _correctly_, you should see your new app running in your _Android Emulator_ or _iOS Simulator_ shortly provided you have set up your emulator correctly.
+
+`yarn android` and `yarn ios` is one way to run your app - you can also run it directly from within Android Studio and Xcode respectively.
+
+
+## Step 3: Modifying your App
+
+Now that you have successfully run the app, let's modify it.
+
+1. Open `App.tsx` in your text editor of choice and edit some lines.
+2. For _Android_: Press the <kbd>R</kbd> key twice or select **"Reload"** from the **Developer Menu** (<kbd>Ctrl + M</kbd> or <kbd>⌘ + M</kbd>) to see your changes!
+   
+   For _iOS_: Hit <kbd>⌘ + R</kbd> in your iOS Simulator to reload the app and see your changes!
+
+That's it!
+Congratulations! :tada: You've successfully run and modified your first React Native app. :partying_face:
+
+### Now what?
+
+- If you want to add this new React Native code to an existing application, check out the [Integration guide](https://reactnative.dev/docs/integration-with-existing-apps).
+- If you're curious to learn more about React Native, check out the [Introduction to React Native](https://reactnative.dev/docs/getting-started).
```

Comment:
This is all copied directly from the website. Judging by how folks have been mentioning tweaks to this already, it seems like having this second copy is likely to cause things to go out of sync as instructions change and are clarified. Maybe we should just link instead?

Reply:
>Though not direct copy-pasta..

Actually, these are ***generic*** steps and if I can predict furture correctly, the commands and steps will remain same atleast for few _major_ versions.

Another thing, the purpose of _such_ READMEs is to help a developer, within _IDE itself_.

At least; **"I won't prefer switching to website tab opened in bowser window"**. When I can simply have the REDME opened with _same_ steps in the IDE itself.

Reply:
Instead, I think; a _Note_ at the start (before Getting Started heading) would be fine, like:

>**NOTE**: You can follow the most up-to-date instructions from the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup?guide=native) page.

Reply:
We can check that assumption by looking at the commit history of the website document this is referencing. https://github.com/facebook/react-native-website/commits/main/docs/getting-started.md

We edit it roughly once a month.